lib/teen_process.js

```diff
@@ -100,6 +100,11 @@ class SubProcess extends EventEmitter {
     this.opts = opts;
   }
 
+  get isRunning () {
+    // presence of `proc` means we have connected and started
+    return !!this.proc;
```

Comment:
Hmm. True... But to my mind "running" means that the process has been kicked off. It is running even if it hasn't "started" from the perspective of the using code.